lib/dat-worker-pool.rb

```diff
@@ -51,16 +51,8 @@ def add_work(work_item)
     @queue.push work_item
   end
 
-  def num_workers
-    @runner.num_workers
-  end
-
-  def waiting
-    @runner.workers_waiting_count
-  end
```

Comment:
@jcredding it would be great to get an `available_worker_count` or something like that method on both the worker pool and the runner.  I'd think it could have utility and seems pretty cheap to offer, right?